### PR TITLE
First draft Jupyter notebook docs

### PR DESCRIPTION
**Pull request by jleibs**
Note some of the links in here aren't live yet -- they point to `latest` rather than `HEAD` but will be correct once Release 0.5.0 is out.

Build preview from:
 - https://github.com/rerun-io/landing/pull/281
 - https://deploy-preview-281--beep-boop-seeing-robots.netlify.app/docs/howto/notebook

### Checklist
* [ ] I have read and agree to [Contributor Guide](../CONTRIBUTING.md) and the [Code of Conduct](../CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/howto/notebook.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Because the Rerun viewer in the notebook is just an embedded HTML snippet it also works with
+tools like nbconvert.
+
+You can convert the notebook to HTML using the following command:
+```
+$ jupyter nbconvert --to=html --ExecutePreprocessor.enabled=True examples/python/notebook/cube.ipynb
+```
+
+This will create a new file `cube.html` that can be hosted on any static web server.
```
Comment (emilk) [inline docs/howto/notebook.md:84]: We should link to such a `cube.html` here that we host. We could just upload it manually for now, but adding it to the CI on publish would be even better.

Comment (emilk) [reply docs/howto/notebook.md:84]: Ah yes: https://github.com/rerun-io/rerun/issues/1890

**Review by emilk — APPROVED** [review]
This is great! I had no problem following along.

**Review by Wumpf — APPROVED** [review]
took this as an opportunity to try stuff out a tiny little bit and omg so cool. Looking forward to sharing these things in our release notes/noise

File: docs/howto/notebook.md (hunk 1 above)
Comment (jleibs) [reply docs/howto/notebook.md:84]: Great idea: I think I can add a sample to docs-media manually.

File: docs/howto/notebook.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+As of version 0.5.0, Rerun now has limited support for embedding the Rerun viewer directly within IPython-style
+notebooks.  This makes it easy to iterate on API calls as well as to share data with others.
+
+Rerun has been tested with:
+ - [Jupyter Notebook Classic](https://jupyter.org/)
+ - [Jupyter Lab](https://jupyter.org/)
+ - [VSCode](https://code.visualstudio.com/blogs/2021/08/05/notebooks)
+ - [Google Colab](https://colab.research.google.com/)
```
[the remark this answers is not in the document]
Comment (jleibs) [reply docs/howto/notebook.md:14]: Our current approach is quite portable, but the future stuff I want to do with incremental updating over notebook widget APIs will be a lot less portable. We should definitely add this stuff to our release-testing checklist.

**Pull request merged**